test/routes.spec.js

```diff
@@ -28,4 +52,45 @@ describe('Favorite Routes', () => {
       done();
     })
   })
+  it('should return a 404 for a route that does not exist', done => {
```

Comment:
Sad path testing looks good! Like we chatted about, it seems like the best option is to restrict the 1-100 rating parameter on the front end when a person can update a favorite!